src/app/modules/shared/components/breadcrumb/breadcrumb.model.ts

```diff
@@ -0,0 +1,8 @@
+import { Route } from '@angular/router'
+
+export class Breadcrumb {
```

Comment:
Instead of defining this in its own file as a class, we should use a typescript interface such as:
```js
interface Breadcrumb {
  displayName: string, 
  url: string, 
  route?: Route
}
```